docs/conf.py

```diff
@@ -350,3 +351,6 @@
     "pillow-logo-dark-text-1280x640.png"
 )
 ogp_image_alt = "Pillow"
+
+# sphinx-reredirects
+# redirects = {"installation.html": "installation/index.html"}
```

Comment:
If this is commented out, then we aren't actually using sphinx-reredirects, right? We could remove it, like https://github.com/radarhere/Pillow/commit/7aba72a48f1b55a3716409f66e772af740f40616?